### PR TITLE
Update partner listing on home page

### LINKED ISSUE #1906: Update example site references at the bottom of the home page (10/19)
We have 6 example sites that run on cloud.gov at the bottom of our homepage, and at least one of them has been decommissioned and is no longer live.  We need to update this list/table to make sure we have recent, relevant examples.

## Acceptance Criteria
* [x] GIVEN links to government websites listed on our homepage
  WHEN a user clicks on any of them
  THEN they are taken to a working site

--- 

## Security considerations
- None

## Implementation sketch
* [x] Work with BU/customer engagement to determine which sites to link to; might need to reach out to the client first for permission

### PR DESCRIPTION
**Pull request by mheadd**
## Changes proposed in this pull request:
- Update sample partner sites at bottom of home page.
- Switches out "Every Kid in a Park" (DOI) for "Dietary Supplement Label Database" (HHS/NIH)
- Fixes #1906 

:sunglasses: [PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/mheadd-patch-20)

## Security Considerations
None

### CODE REVIEW

**Comment by mheadd** [comment]
@kelleyconfer Thoughts on whether we would need to ask NIH if we can list them in our customer section?

**Review by apburnes — APPROVED** [review]
🍏 🥕 🥬

**Comment by mheadd** [comment]
Waiting on a response to this before merging: https://cloud-gov.zendesk.com/agent/tickets/3716

**Pull request merged**
